recipes/_install_rbenv.rb

```diff
@@ -1,10 +1,14 @@
 include_recipe 'ruby_rbenv::default'
-include_recipe 'ruby_rbenv::system_install'
 include_recipe 'ruby_build::default'
 
-rbenv_ruby node['samson']['ruby']['version'] do
-  user node['samson']['user']
-end
+node.default['rbenv']['user_installs'] = [
```

Comment:
Well renaming samson.rb to default.rb would seem to be much simpler while also ensuring that everything's where we expect it to be, no ?